test/check-application

```diff
@@ -145,6 +145,8 @@ class TestApplication(testlib.MachineCase):
 
         self.allow_journal_messages("/run.*/podman/podman: couldn't connect.*")
         self.allow_journal_messages(".*/run.*/podman/podman.*Connection reset by peer")
+        # Ubuntu 22.04 doesn't support user podman-systemd-generator
+        self.allow_journal_messages("Generator directory or unit not found:.*")
```

Comment:
Please also match on the `/run/user/.*/` path to ensure it's the user generator, not the system one.